diff --git a/node_modules/zustand/middleware/persist.d.ts b/node_modules/zustand/middleware/persist.d.ts
index acb6de5..6bd7631 100644
--- a/node_modules/zustand/middleware/persist.d.ts
+++ b/node_modules/zustand/middleware/persist.d.ts
@@ -93,7 +93,7 @@ declare type Cast<T, U> = T extends U ? T : U;
 declare type WithPersist<S, A> = S extends {
     getState: () => infer T;
 } ? Write<S, StorePersist<Cast<T, State>, A>> : never;
-interface StorePersist<S extends State, Ps> {
+export interface StorePersist<S extends State, Ps> {
     persist: {
         setOptions: (options: Partial<PersistOptions<S, Ps>>) => void;
         clearStorage: () => void;
